vulkan_generator/codegen_utils/test.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (C) 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This module contains some cheap and simple tests for the code generation"""
+
+from codegen_utils import codegen as codegen_utils
+
+print("=== Codegen Tests ===")
+
+signature = codegen_utils.create_function_signature("fib", "unsigned int",
```

Comment:
These lines could be 

```
f"""
// generate the Nth number of the fibonacci sequence
unsigned int fib(unsigned int);
"""
```

which is much more readable than looking at the generator functions.

Reply:
This file was scaffolding from when I was developing the codegen, so I could see what my functions were generating. There are now many unit tests for the generated code using gtest